extensions/2.0/Vendor/EXT_feature_metadata/schema/featureCollection.property.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Property array",
+  "type": "object",
+  "description": "Array containing property data",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "elementCount": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "The number of elements in the array. `elementCount` may be greater than `featureCount` if the array contains extra elements not used by this model, e.g. if different models refer to different sections of a global array or if the arrays represents a larger data dictionary."
+      },
+      "elementByteLength": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "A fixed-size byte length for `STRING` and `BINARY` elements. When omitted, elements are variable-size."
+      },
+      "bufferView": {
+          "allOf": [ { "$ref": "glTFid.schema.json" } ],
+          "description": "The index of the `bufferView` containing property array data. The bufferView must be aligned to a multiple of 8 bytes. For a GLB file, this is measured relative to the beginning of the file. For a glTF + BIN file, this is relative to the beginning of the BIN file. To accomplish this, padd each bufferView to 8-bit alignment using 0 (NUL) characters."
+      },
+      "offsetsBufferView": {
+          "allOf": [ { "$ref": "glTFid.schema.json" } ],
+          "description": "The index of the `bufferView` containing offsets to elements. Required for `VARIABLE_LENGTH_ARRAY`, `BINARY` or `STRING` element types. The offsets buffer contains `elementCount + 1` unsigned integers (TODO: 32-bit or 64-bit, or configurable), which represent the start position of each element in the buffer, with the last offset representing the position after the last element. The byte length of strings, binary blobs, or element arrays is computed using the difference between the current offset and the subsequent offset."
```

Comment:
@lilleyse or @baothientran any opinion on whether the offsets buffer should have configurable element size (uint32 or uint64)?

I vote configurable.

Reply:
I vote configurable as well. If we go that route we should support all unsigned int types: uint8, uint16, uint32 (default), or uint64.

How does this impact the schema? Just another property "offsetComponentType"?

Reply:
I would call it `offsetElementType` since the offset is a scalar type, not a vec3 or string (just for consistency). Otherwise I agree.

Reply:
I vote configurable as well. It will support for larger size if user needs

Reply:
I updated the schema here to allow for this.

Another case where this would be helpful is if the user wants to pack things in tighter to save space, jumping down to `uint8` or `uint16` would be helpful too.